sap_business_document_processing/document_information_extraction_client/constants.py

```diff
@@ -6,17 +6,38 @@
 API_FIELD_CLIENT_LIMIT = 'limit'
 API_FIELD_CLIENT_NAME = 'clientName'
 API_FIELD_DOCUMENT_TYPE = 'documentType'
+API_FIELD_DOCUMENT_TYPE_DESCRIPTION = 'documentTypeDescription'
 API_FIELD_ENRICHMENT = 'enrichment'
 API_FIELD_EXTRACTED_HEADER_FIELDS = 'headerFields'
 API_FIELD_EXTRACTED_LINE_ITEM_FIELDS = 'lineItemFields'
 API_FIELD_EXTRACTED_VALUES = 'extractedValues'
 API_FIELD_FILE_TYPE = 'fileType'
+API_FIELD_FILTER = 'filter'
 API_FIELD_ID = 'id'
+API_FIELD_PREDEFINED = "predefined"
+API_FIELD_PROPERTIES = 'properties'
 API_FIELD_RESULTS = 'results'
 API_FIELD_RETURN_NULL = 'returnNullValues'
 API_FIELD_SCHEMA_ID = 'schemaId'
+API_FIELD_SCHEMA_NAME = 'name'
```

Comment:
This constants might be misleading since it sounds as if the string would be 'schemaName'. Maybe rename to `API_FIELD_NAME`

Reply:
Updated.